pkg/cmd/config.go

```diff
@@ -457,6 +461,7 @@ func newConfig(options ...configOption) (*Config, error) {
 		"keeper":                   c.keeperTemplateFunction,
 		"keeperDataFields":         c.keeperDataFieldsTemplateFunction,
 		"keeperFindPassword":       c.keeperFindPasswordTemplateFunc,
+		"ph":                       c.passholeTemplateFunction,
```

Comment:
All other password manager template functions use the full name of the password manager. This function should be called `passhole`, not `ph`.